doc/UsersGuide/BackgroundInfo/TechnicalOverview.rst

```diff
@@ -183,28 +183,42 @@ The ``ufs-srweather-app`` :term:`umbrella repository` is an NCO-compliant reposi
    │     ├── bash_utils
    │     ├── machine
    │     ├── wrappers
+   │     ├── python_utils
    │     ├── config.community.yaml
+   │     ├── config.*.yaml
    │     ├── config_defaults.yaml
-   │     ├── experiment.jsonschema
```

Comment:
This file (and probably `user.jsonschema`) should remain here--they came into develop with the UW Tools PR. I'd recommend rechecking what is currently in the `ush` directory and making sure that any old files are removed (e.g., `valid_param_vals.yaml`) and new ones are added so that this diagram reflects the actual state of `develop` branch `ush`.